docs/source/changelog.rst

```diff
@@ -7,7 +7,7 @@ Changelog
 Array
 +++++
 
-- Allow slicing a Dask Array by another one-dimensional Dask Array of integers
+-
```

Comment:
you can probably add this back? :)

Reply:
I removed to reduce merge collisions, in accordance to the new policy of the dask project